server/sockets/list.js

```diff
@@ -278,12 +282,16 @@ const addOwnerRoleInList = (socket, clients) => {
       });
 
     if (clients.has(userId)) {
-      socket.broadcast
-        .to(clients.get(userId))
-        .emit(ListActionTypes.ADD_OWNER_ROLE_SUCCESS, {
-          ...data,
-          isCurrentUserRoleChanging: true
-        });
+      const { viewId, socketId } = clients.get(userId);
+
+      if (viewId === listId) {
+        socket.broadcast
+          .to(socketId)
+          .emit(ListActionTypes.ADD_OWNER_ROLE_SUCCESS, {
+            ...data,
+            isCurrentUserRoleChanging: true
+          });
+      }
```

Comment:
the same here, check previous comment

Reply:
we have to first check if exist then destructure to certain props as we discussed